index.js

```diff
@@ -151,6 +152,7 @@ export function execa(rawFile, rawArgs, rawOptions) {
 	}
 
 	const controller = new AbortController();
+	setMaxListeners(Number.POSITIVE_INFINITY, controller.signal);
```

Comment:
One `AbortController` is created per `execa()` call and we're the only one using it, so it's pretty safe to undo the `maxListeners` warning altogether.

In principle, we could be stricter and compute the exact amount of times `signal.on('abort')` is being used, but that's fairly tedious and would not be give much benefits.